javascript-data-model/order-history.js

```diff
@@ -0,0 +1,71 @@
+
+var orderHistory = [
+  {
+    shipTo: 'JS Masher',
+    orderPlaced: 'August 4, 2020',
+    deliveryDate: 'August 8, 2020',
+    orderNumber: '114-3941689-8772232',
+    returnWindow: 'September 7, 2020',
+    total: 34.00,
+    items: [
+      {
+        title: 'JavaScript for impatient programmers',
+        author: 'Rauschmayer, Dr. Axel',
+        price: 31.55
+      }
+    ]
+  },
+  {
+    shipTo: 'JS Masher',
+    orderPlaced: 'July 19, 2020',
+    deliveryDate: 'July 20, 2020',
+    orderNumber: '113-9984268-1280257',
+    returnWindow: 'August 19, 2020',
+    total: 44.53,
+    items: [
+      {
+        title: 'The Timeless Way of Building',
+        author: 'Alexander, Christopher',
+        price: 41.33
+      }
+
+    ]
+
+  },
+  {
+    shipTo: 'JS Masher',
+    orderPlaced: 'August 5, 2020',
+    deliveryDate: 'July 7, 2020',
+    orderNumber: '114-2875557-9059409',
+    returnWindow: 'August 5, 2020',
+    total: 17.22,
+    items: [
+      {
+        title: 'Gamecube Controller Adapter',
+        author: NaN,
```

Comment:
`NaN`? How about `null`?

Reply:
Forgot to change that to something better later lmao